src/components/StudentGroup/index.tsx

```diff
@@ -159,21 +169,54 @@ const StudentGroup = observer((props: Props) => {
                     <dt>Anzahl Schüler:innen</dt>
                     <dd>
                         <span className={clsx('badge badge--primary')}>
-                            {group.students.filter((u) => !u.isAdmin).length}
+                            {group.students.filter((u) => !u.hasElevatedAccess).length}
                         </span>
                     </dd>
-
-                    <dt>Lehrpersonen</dt>
-                    <dd>
-                        <ul>
-                            {group.students
-                                .filter((u) => u.isAdmin)
-                                .map((user, idx) => (
-                                    <li key={idx}>
-                                        {user.firstName.slice(0, 1)}. {user.lastName}
+                    <dt>Admins</dt>
+                    <dd className={clsx(styles.ddGroup)}>
+                        <div className={styles.listContainer}>
+                            <ul className={clsx(styles.students, styles.list)}>
+                                {group.admins.map((admin, idx) => (
+                                    <li key={idx} className={clsx(styles.listItem)}>
+                                        {admin.nameShort}
+                                        {isAdmin && (
```

Comment:
A tiny detail: The confirmation boxes for both "Adminrecht entziehen?" and "Admin aus der Gruppe entfernen" can be open at the same time, which is a bit messy. It would be fancy if only one of them could be open at a time, and the other would just collapse automatically. But this really isn't important ;)

Reply:
Hihi, indeed this would be really cool. Anyway, this would need quiet a bit of refactoring. We can take this to our backlog ;)
https://github.com/GBSL-Informatik/teaching-dev/issues/142